CHANGELOG.md

```diff
@@ -44,6 +44,17 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - Switched from port `53` to port `1053` for coredns inside container.
 
+__Warning__: This change means the default port `53` is blocked as it is a priviledge port. In case of using custom block (`coredns-user-values`) you need to update them to specify the port `1053` like in this example
```

Comment:
```suggestion
__Warning__: This change is because the default port `53` is blocked because it is a privileged port. In case you are using the custom block (`coredns-user-values`) you need to update it to specify the port `1053` like in this example.
```